parm/aero/berror/aero_diffusionparm.yaml.j2

```diff
@@ -34,7 +34,7 @@ background error:
   saber central block: 
     saber block name: diffusion
     calibration:
-      normalization:
```

Comment:
Why add these extra spaces?

Reply:
Do you mean the indentation of normalization: ?   If so, I have it working with this spacing.  Should normalization be at the same indenation as calibration?


> Why add these extra spaces?

Do you mean the indentation of normalization: ?   If so, I have it working with this spacing.  Should normalization be at the same indentation as calibration?

Reply:
No I mean the PR includes blank spaces after the :